databind/src/main/java/tech/ydb/yoj/databind/StringValueConverter.java

```diff
@@ -0,0 +1,106 @@
+package tech.ydb.yoj.databind;
```

Comment:
suggestion: maybe move it to separate package like tech.ydb.yoj.databind.converter ?

Reply:
Good idea, moved `*Converter` classes to a separate package! :+1: